jujugui/static/gui/src/app/components/profile/profile.js

```diff
@@ -0,0 +1,80 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+class Profile extends React.Component {
+
+  constructor(props) {
+    super(props);
+    const activeSection = this.props.activeSection || 'models';
+    this.state = {activeSection};
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({activeSection: nextProps.activeSection});
+  }
+
+  shouldComponentUpdate(nextProps, nextState) {
+    if (nextProps.activeSection !== this.state.activeSection) {
```

Comment:
This function could be rewritten as just `return this.state.activeSection !== nextProps.activeSection;`